### DISCUSSION TITLE
Camera Splitscreen tangled

### DISCUSSION THREAD

**Discussion by lurgx** [godotengine/godot-proposals#4332] [2D]
Although the split screen can be done, the truth is that it is somewhat complicated to make splitscreen for both players, it would be good if it were simple as in 3d, this led me to create a camera like in metalslug where the camera follows both players only if both are in the edge that has been great it would be good if that could also be implemented to the camera since it is also something very laborious, of course it can be understood if they have more work for which we will continue supporting them and we hope to continue donating and not just demanding :)
Now it would also be nice to be able to use the 3D camera as the 2d camera with its limits in any Vector
I was developing a megaman style game but in 3D but the camera did not offer me the properties of the 2d camera such as how much it must be on the edge to advance the camera
 or the limits
It would also be great if they could implement this that would allow 3d games with 2d view style Megaman 11
Although it can be done with areas or by code, it is quite messy and frustrating at times
Thanks
2d Type 3d Camera Demonstration Video:
https://www.youtube.com/watch?v=_7TXvfWVFNw

**Comment by Calinou** [comment]
This can already be done. There's an official demo showcasing this: https://github.com/godotengine/godot-demo-projects/tree/master/viewport/dynamic_split_screen

**Reply by lurgx** [reply]
Great even so I still believe that it is easier to do split screen in 3d as opposed to 2d I have also seen your split screen page :D
